manager/assets/modext/widgets/core/modx.window.js

```diff
@@ -197,6 +322,60 @@ Ext.extend(MODx.Window,Ext.Window,{
                 }
             }
         }
+
+        /*
+            When a switch is rendered in the footer bar, we need to
+            insert a hidden field in the form to to be able to relay its value to
+            the processor
+        */
+        if (this.config.hasOwnProperty('modxFbarSaveSwitches') && this.config.modxFbarSaveSwitches.length > 0) {
+            this.config.modxFbarSaveSwitches.forEach(saveSwitch => {
+                switch (saveSwitch) {
+                    case 'redirect':
+                        defaultValue = this.config.redirect ;
+                        break;
+                    case 'duplicateValues':
+                        defaultValue = 0;
+                        break;
+                    default:
+                        defaultValue = 1;
+
+                }
+                this.setFbarSwitchHiddenField(saveSwitch, defaultValue);
+            });
+
+        }
+        console.log('final fields: ', this.config.fields);
```

Comment:
stray log and a bunch of commented out code below that can be removed (or is that still needed to be implemented?)